qualtran/cirq_interop/cirq_interop.ipynb

```diff
@@ -455,55 +455,6 @@
     "out_quregs"
    ]
   },
-  {
-   "cell_type": "markdown",
-   "id": "896e0392-3f1f-4965-bb13-0a541581ec25",
-   "metadata": {},
-   "source": [
-    "## Test `Cirq -> Bloqs -> Cirq` roundtrip using `QROM`\n",
```

Comment:
is this removal on purpose?

Reply:
Yeah, we've come a long way since this was written. The cirq interop probably needs to be updated now but maybe as a follow up. 


The idea is that now `BloqAsCirqGate` should be used to wrap only bloqs which are not already cirq gates (I've added an assertion to the constructor). `QROM` is a `GateWithRegisters` so should never be wrapped in a `BloqAsCirqGate`. This section was written when `QROM` was implemented in Cirq-FT and was not already a Bloq. 

Other examples of wrapping cirq-only gates in `CirqGateAsBloq` and bloq-only bloqs in `BloqAsCirqGate` are still part of the notebook.